src/main/java/org/jenkinsci/plugins/casc/plugins/PluginManagerConfigurator.java

```diff
@@ -27,15 +27,7 @@
 import java.io.File;
 import java.io.IOException;
 import java.io.PrintWriter;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.LinkedList;
-import java.util.List;
-import java.util.Map;
-import java.util.Queue;
-import java.util.Set;
+import java.util.*;
```

Comment:
please configure your IDE to not use star imports 😭

Reply:
Done...i always just used the default setting in Intellij. What is the justification for not using the default? Doesn't the compiler optimize everything not needed away or is just for clarity this is done?

Reply:
https://stackoverflow.com/questions/147454/why-is-using-a-wild-card-with-a-java-import-statement-bad covers it pretty well :) From a Jenkins code standard they also advice against it. If I didn't Oleg would have 😄